pkgs/development/tools/summon/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, buildGoModule, fetchFromGitHub, lib, patchResolver ? true }:
+
+with stdenv.lib;
+
+buildGoModule rec {
+  pname = "summon";
+  version = "0.8.1";
```

Comment:
Version 0.8.2 is available upstream:
https://github.com/cyberark/summon/releases/tag/v0.8.2

Reply:
Resolved, thanks.